index.bs

```diff
@@ -2739,7 +2773,7 @@ attributes.
 
         The [=[RP]=] can determine the resulting [=authenticator attachment modality|attachment=] for the created credential
         using the {{AuthenticatorAttestationResponse/getTransports()}} method of the resulting {{AuthenticatorAttestationResponse}}.
```

Comment:
```suggestion
        The authenticator's [=authenticator attachment modality|attachment modality=] for the created credential
        is conveyed in {{PublicKeyCredential/authenticatorAttachment}}.
```

Reply:
This was made obsolete by rebasing onto #1668 as the affected language no longer remains.